ush/machine/orion.yaml

```diff
@@ -19,8 +19,6 @@ platform:
   RUN_CMD_POST: srun
   RUN_CMD_SERIAL: time
   RUN_CMD_UTILS: srun
-  ENV_INIT_SCRIPTS_FPS:
```

Comment:
Should the TOPO_DIR and SFC_CLIMO_INPUT_DIR entries be moved to task_make_orog and task_make_sfc_climo like the rest of the machine yaml files?